src/app/material/material.module.ts

```diff
@@ -17,6 +17,7 @@ import {
   MatSortModule
 } from "@angular/material";
 import { CdkTableModule } from "@angular/cdk/table";
+import { ChartsModule } from "ng2-charts";
```

Comment:
not sure about `ChartsModule`. this is not material, so it might have to be directly in `transactions.module`

Reply:
this statement conflicts with previous one as you told about moving ChartsModule from app module to material module